### PR TITLE
tests tools: add prometheus metrics

### PR DESCRIPTION
**Pull request by fanatid**
@linuskendall are these metrics what you wanted?

### CODE REVIEW

File: tests/bgtask_creator/src/main.rs
Comment (linuskendall) [inline tests/bgtask_creator/src/main.rs:1]: Add prom to `create` command too, that shows the number of tasks created.

File: tests/tree-status/src/main.rs
Comment (linuskendall) [inline tests/tree-status/src/main.rs:1]: Metrics:

tree_status_completed_trees
tree_status_incomplete_trees

maybe:
tree_status_max_seq{tree="abc"} = <seq>
tree_status_missing_seq{tree="abc"} = <number of missing sequences>

File: tests/bgtask_creator/src/main.rs
Comment (linuskendall) [reply tests/bgtask_creator/src/main.rs:1]: Maybe add a label for collection

tasks_created{collection="<abc>"}
tasks_created{creator="<bed>"}
tasks_created{} # if no filter was specified

**Review by linuskendall — APPROVED** [review]
looks good,

added a few suggestions about additional metrics

we could also add metrics to txn_forwarder / acc_forwarder (just i ugess number of transactions forwarded / number of accounts forwarded?)

File: tests/tree-status/src/main.rs
Comment (fanatid) [reply tests/tree-status/src/main.rs:1]: hm, how you want to implement `tree_status_missing_seq{tree="abc"} = `? We have array of missing sequences, we can not put it to counter / gauge :confused:

**Comment by fanatid** [comment]
Updated

File: tests/tree-status/src/main.rs
Comment (linuskendall) [reply tests/tree-status/src/main.rs:1]: just the count of the array

**Pull request merged**